scheduler/scripts/microbenchmarks/sweep_models_for_overhead.py

```diff
@@ -0,0 +1,124 @@
+import os, sys
+sys.path.append(os.path.dirname(os.path.dirname(os.path.dirname(os.path.realpath(__file__)))))
+
+import argparse
+import asyncio
+import numpy as np
+import re
+import time
+
+from job import Job
+from job_id_pair import JobIdPair
+from job_table import JobTable
+
+BASE_SCHEDULER_COMMAND = \
+    """python scripts/drivers/run_scheduler_with_trace.py \
+        --seed 0 \
+        --solver ECOS \
+        --throughputs_file physical_cluster_throughputs.json \
+        --time_per_iteration 360 \
+        --policy max_min_fairness \
+        --expected_num_workers 1"""
+BASE_WORKER_COMMAND = \
+    """python worker.py -i 127.0.1.1 -s 50060 -w 50061 -g 1 -t p100"""
+INFINITY = 10000000000
+
+def get_overhead(line):
+    pattern = ('Job (\d+): Total time=([-+]?[0-9]*\.?[0-9]+) seconds, '
+               'Computation time=([-+]?[0-9]*\.?[0-9]+) seconds, '
+               'Overhead=([-+]?[0-9]*\.?[0-9]+)%')
+    match = re.match(pattern, line)
```

Comment:
Where is this overhead being computed?

Reply:
Like this is just reading some file?

Reply:
Ah yea I moved the actual overhead computation to the other script, I'll get rid of this code